Billing.API/Program.cs

```diff
@@ -36,6 +36,13 @@ public static void Main(string[] args)
 
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration((hostingContext, config) =>
+                {
+                    config.AddJsonFile("appsettings.Secret.json", true);
+                    config.AddJsonFile("/run/secrets/appsettings.Secret.json", true);
+                    config.AddKeyPerFile("C:\\ProgramData\\docker\\secrets", true);
+                    config.AddKeyPerFile("/run/secrets", true);
```

Comment:
Based on https://github.com/MakingSense/doppler-jobs/pull/46

```suggestion
                    config.AddJsonFile("appsettings.Secret.json", true);
                    config.AddJsonFile("C:\\ProgramData\\Docker\\secrets\\appsettings.Secret.json", true);
                    config.AddJsonFile("/run/secrets/appsettings.Secret.json", true);
                    config.AddKeyPerFile("C:\\ProgramData\\Docker\\secrets", true);
                    config.AddKeyPerFile("/run/secrets", true);
```